pkg/api/kyverno/v1/policy_types.go

```diff
@@ -150,7 +150,9 @@ type Condition struct {
 	// +kubebuilder:validation:XPreserveUnknownFields
 	Key apiextensions.JSON `json:"key,omitempty" yaml:"key,omitempty"`
 
-	// Operator is the operation to perform.
+	// Operator is the operation to perform. Valid operators
+	// are Equals, NotEquals, In and NotIn.
+	// +kubebuilder:validation:XPreserveUnknownFields
```

Comment:
@realshuting - Is this what you were referring to when you talked about [adding validation checks](https://github.com/kyverno/kyverno/issues/1584#issuecomment-778529134)?